spec/controllers/idv/phone_controller_spec.rb

```diff
@@ -69,4 +61,12 @@
       end
     end
   end
+
+  def prime_subject(user)
```

Comment:
Maybe a better method name would be something like `stub_before_actions` or `stub_idv_setup`, or something along those lines?